en_ewt-ud-test.conllu

```diff
@@ -13697,7 +13697,7 @@
 # text = Play your CD's, 8-tracks, reel to reels, cassettes, vinyl 33/45's, and shellac 78's through this little integrated amp, you'll get a big eye opener !
 1	Play	play	VERB	VB	Mood=Imp|VerbForm=Fin	0	root	0:root	_
 2	your	your	PRON	PRP$	Case=Gen|Person=2|Poss=Yes|PronType=Prs	3	nmod:poss	3:nmod:poss	_
-3	CD's	cd'	NOUN	NNS	Number=Plur	1	obj	1:obj	SpaceAfter=No
+3	CD's	cd	NOUN	NNS	Number=Plur	1	obj	1:obj	SpaceAfter=No
```

Comment:
CD

Reply:
When not as a proper noun, `CD` has been lemmatized as `cd` elsewhere in EWT.  For example:

```
# sent_id = newsgroup-groups.google.com_alt.animals_434fe80fb3577e8e_ENG_20031011_200300-0060
14      it      it      PRON    PRP     Case=Nom|Gender=Neut|Number=Sing|Person=3|PronType=Prs  18      nsubj:outer     18:nsubj:outer  _
15      is      be      AUX     VBZ     Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin   18      cop     18:cop  _
16      because because SCONJ   IN      _       18      mark    18:mark _
17      you     you     PRON    PRP     Case=Nom|Person=2|PronType=Prs  18      nsubj   18:nsubj        _
18      own     own     VERB    VBP     Mood=Ind|Number=Sing|Person=2|Tense=Pres|VerbForm=Fin   0       root    0:root  _
19      some    some    DET     DT      _       20      det     20:det  _
20      L.P     l.p     NOUN    NN      Number=Sing     18      obj     18:obj  SpaceAfter=No
21      ,       ,       PUNCT   ,       _       22      punct   22:punct        _
22      CD      cd      NOUN    NN      Number=Sing     20      conj    20:conj:or      _
```